src/Transformers/Family.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Choredo\Transformers;
+
+use Assert\Assertion;
+use Choredo\Entities;
+use League\Fractal\TransformerAbstract;
+
+class Family extends TransformerAbstract
+{
+    const DAYS_OF_WEEK = ['sunday', 'monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday'];
```

Comment:
This already exists in Schedule, can we keep it in one place or the other? Which feels most correct?

Reply:
Removing it from Schedule. It makes sense in the transformer since it's purely presentation logic. Internally the data is managed as an integer.